### PR TITLE
finalizing-setup: add note if GM9 fails to start

### PR DESCRIPTION
**Pull request by ihaveamac**
I found someone who misspelled "payloads" as "playloads", causing
confusion for a while until I saw a directory listing. I don't know how
common exactly this is but I feel a note could be added here.

I'm not too happy about how this is worded though. Especially the path
which might need to be updated following #1829.

### CODE REVIEW

**Comment by lilyuwuu** [comment]
"repeated attempts" is kind of vague. I'd suggest rewording the instructions to prevent any chance of holding START too late, and to declaratively say if it doesn't work to check file placement.


1. Power off your device
2. Hold the (Start) button
3. Power on your device while continuing to hold the (Start) button
4. If you were successful, you should boot into GodMode9
    - If you boot to the home menu, ensure that `GodMode9.firm` is in `luma -> payloads` and that `payloads` is properly spelled
5. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has completed

There's probably a better way to word this too.

**Comment by MechanicalDragon0687** [comment]
I agree with adding the "Hold start" first, to prevent someone pressing start after power
I think paths should be consistent throughout the guide, or at least the page. Changing pattern to `luma -> payloads` is inconsistent with the current pattern.

**Comment by lilyuwuu** [comment]
Changing pattern is a potential plan based on #1829. Should probably be a separate PR though since it'd change nearly every page.
